workers/sendWeeklyRecapEmails.js

```diff
@@ -161,12 +161,21 @@ var worker = function(job, done){
 
             variables.WEAK_HABITS = 0;
             variables.STRONG_HABITS = 0;
+            // minimum value a negative-only habit has to reach to be considered strong
+            var weakHabitCutoff = -1;
 
             user.habits.forEach(function(habit){
-              if(habit.value < 1){
-                variables.WEAK_HABITS++;
-              }else{
+              // treat brightly yellow negative-only habits as strong
+              if(habit.down == true && habit.up == false && habit.value > weakHabitCutoff){
```

Comment:
should this be `>= weakHabitCutoff` ? 

A value of exactly -1 is neutral (yellow). https://github.com/HabitRPG/habitrpg/blob/dcd7cae31295566b8fd08e5a8cec076f975b45b0/common/script/index.js#L396-L410